### ISSUE TITLE
Automation only has enable on/off as "action" to controll charger

### ISSUE THREAD

**Issue by large** [fondberg/easee_hass#153]
Hi!
I currently working on reducing hourly usage in my house and I want to control the Easee charger.
It is setup with a schedule to charge from 22:00 to 06:00, but if the house hourly totals is above a certain limit I want to pause to the next hour.

First version of control is like this :
```
###########################
#Stop charger
- id: '1639602865859'
  alias: 'Garasje: Stoppe lader ved for høyt forbruk'
  description: ''
  trigger:
  - platform: state
    entity_id: sensor.garasjen_status
    to: charging
  condition:
  - type: is_energy
    condition: device
    device_id: 732f06447e131fc50d2e1ecaf673117b
    entity_id: sensor.accumulated_consumption_current_hour_nustadsloyfa_4
    domain: sensor
    above: 4.5
  action:
  - type: turn_off
    device_id: 0a57fd5c74291899b431e9b10c1d0ce4
    entity_id: switch.garasjen_is_enabled
    domain: switch
  mode: restart

###########################
#Start charger again
- id: '1639603157022'
  alias: 'Garasje: Starte lader ved begynnelse på en ny time'
  description: ''
  trigger:
  - type: energy
    platform: device
    device_id: 732f06447e131fc50d2e1ecaf673117b
    entity_id: sensor.accumulated_consumption_current_hour_nustadsloyfa_4
    domain: sensor
    above: 0
    below: 4.5
  condition:
  - condition: device
    type: is_off
    device_id: 0a57fd5c74291899b431e9b10c1d0ce4
    entity_id: switch.garasjen_is_enabled
    domain: switch
  action:
  - type: turn_on
    device_id: 0a57fd5c74291899b431e9b10c1d0ce4
    entity_id: switch.garasjen_is_enabled
    domain: switch
  mode: restart
```

It seems quite "odd" to enable/disable the charger to pause/resume the charger.
When disabled the charger was greyed out in the app with an "active" button on top. Never seen that before.
Is this the correct way to control this?

Note:
I know that this charger is much "smarter" than this!
I do not need any of the load balancing and priceadjusted price-algos.
Just a simple pause/resume when connected.

**Comment by tmjo** [comment]
Hi! In fact you answer your own question here: You should not use enable/disable, you could use pause/resume which is available as a service to call (`easee.pause` and `easee.resume`). Another solution can be to control and set the charger dynamic current (or circuit current for that matter, but it would be the same if you have only one charger). Also this is available as service `easee.set_charger_dynamic_limit`. If the dynamic limit is set to zero, the charger is effectively paused and will resume if the limit is increased (it has to be above 6A I believe to open up if idle_current is enabled for heating and such). 

You can read more about the [API here](https://developer.easee.cloud/docs) which explains how it works on the Easee side.

**Comment by large** [comment]
@tmjo thank you for the solution! The service calls was something I never used in HA before.

With the smart `easee.set_charger_dynamic_limit` I can try to use the charger to balance off the house with realtime data through the HAN-port. But i doubt it will work before Easee gets their finger out for a local api https://feedback.easee.com/b/ideas/local-simple-api/

**Comment by tmjo** [comment]
@large: Yes, we have also underlined the importance of local API to Easee when we talked to them. I really hope it will come, but nevertheless many people use the current services with success. Perhaps relying on this to protect the main circuit breaker would not be a good idea, but some 'light' load balancing and for instance to stay below a certain kwh/h or similar is possible. It is probably wise to build the automations smart so that will repeat until the result is achieved (so if offline for a short while, it will still work although delayed).
